.github/actions/nimbus-build-system/action.yml

```diff
@@ -77,15 +86,15 @@ runs:
           printf "'%s'" "$quoted"
         }
 
-        if [[ '${{ inputs.cpu }}' == 'amd64' ]]; then
-          PLATFORM=x64
-        else
-          PLATFORM=x86
-        fi
+        case '${{ inputs.cpu }}' in
+          'amd64') PLATFORM=x64 ;;
+          'arm64') PLATFORM=x64 ;;
```

Comment:
This looks a bit weird, setting the PLATFORM variable to `x64` on an ARM build. It is used to set ARCH_OVERRIDE in nimbus-build-system, and that only checks [whether it equals `x86` or not](https://github.com/search?q=repo%3Astatus-im%2Fnimbus-build-system%20arch_override&type=code). So setting it to `x64` works, but it's not great. Perhaps we can drop this PLATFORM variable all together, and only set ARCH_OVERRIDE when we're on i386?

Reply:
I was not able to find why in general we need `ARCH_OVERRIDE` and who is using it, to set a right value for arm64. I saw just that distinguish between `i386/x64` and ended up with the moving arm64 to x64 side.

Now we set `ARCH_OVERRIDE` only for `i386` and otherwise it will be empty and not passed.